client/src/Clusters/Footer/Footer.style.css

```diff
@@ -0,0 +1,31 @@
+.footer-section {
+    border-top: 1px solid #7f7f7f;
+    padding-top: 30px;
+    width: 90%;
+    display: inline-block;
+}
+
+.footer-left {
+    position: absolute;
```

Comment:
Can you try to use flex?
```
display: flex;
```

https://stackoverflow.com/questions/36182635/making-a-flex-item-float-right/36182782